src/code/z_fbdemo.c

```diff
@@ -27,6 +27,22 @@ Gfx sTransTileSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
+#define SET_VERTEX(vtx, x, y, z, f, s, t, nx, ny, nz, alpha) \
+    {                                                        \
+        Vtx_tn* vtxn = &(vtx)->n;                            \
+        vtxn->ob[0] = (x);                                   \
+        vtxn->ob[1] = (y);                                   \
+        vtxn->ob[2] = (z);                                   \
+        vtxn->flag = 0;                                      \
+        vtxn->tc[0] = (s);                                   \
+        vtxn->tc[1] = (t);                                   \
+        vtxn->n[0] = (nx);                                   \
+        vtxn->n[1] = (ny);                                   \
+        vtxn->n[2] = (nz);                                   \
+        vtxn->a = (alpha);                                   \
+    }                                                        \
+    (void)0
+
```

Comment:
I have an old branch with VTX macros, but I wonder if this could be used elsewhere?
https://github.com/zeldaret/mm/compare/main...engineer124:mm:file_choose_vtx

Reply:
https://github.com/zeldaret/oot/pull/1727#discussion_r1480716622

Maybe but we would need to investigate. Do you know of any other place that sets `vtxn` specifically?

Reply:
Right, I didn't notice it was `Vtx_tn` specifically. It's fine here then